sdk/documentintelligence/Azure.AI.DocumentIntelligence/samples/Sample_AddOnCapabilities.md

```diff
@@ -0,0 +1,747 @@
+# Extract the layout of a document
+
+This sample demonstrates how to extract all identified barcodes using the
+add-on 'Barcodes', 'Formulas', 'FontStyling', 'Languages', 'OcrHighResolution' capabilities.
+
+Add-on capabilities are available within all models except for the Business card
+model. This sample uses Layout model to demonstrate.
+Add-on capabilities accept a list of strings containing values from the `DocumentAnalysisFeature`
+enum class. For more information, see:
+https://learn.microsoft.com/en-us/dotnet/api/azure.ai.formrecognizer.documentanalysis.documentanalysisfeature?view=azure-dotnet
+
+The following capabilities are free:
+- Barcodes
+- Languages
+
+The following capabilities will incur additional charges:
+- Formulas
+- OcrHighResolution
+- FontStyling
+
+See pricing: https://azure.microsoft.com/pricing/details/ai-document-intelligence/.
+
+USAGE:
+
+Set the environment variables with your own values before running the sample:
+1) AZURE_FORM_RECOGNIZER_ENDPOINT - the endpoint to your Form Recognizer resource.
+2) AZURE_FORM_RECOGNIZER_KEY - your Form Recognizer API key 
+
+To get started you'll need a Cognitive Services resource or a Form Recognizer resource.  See [README][README] for prerequisites and instructions.
+
+## Creating a `DocumentAnalysisClient`
+
+To create a new `DocumentAnalysisClient` you need the endpoint and credentials from your resource. In the sample below you'll use a Form Recognizer API key credential by creating an `AzureKeyCredential` object, that if needed, will allow you to update the API key without creating a new client.
+
+You can set `endpoint` and `apiKey` based on an environment variable, a configuration setting, or any way that works for your application.
+
+```C# Snippet:CreateDocumentAnalysisClient
+string endpoint =  Environment.GetEnvironmentVariable("AZURE_FORM_RECOGNIZER_ENDPOINT");
+string apiKey = Environment.GetEnvironmentVariable("AZURE_FORM_RECOGNIZER_KEY");
+var client = new DocumentAnalysisClient(new Uri(endpoint), new AzureKeyCredential(apiKey));
+```
+
+## High resolution extraction
+To extract content from documenta given file at a URI with the high resolution capability, use the `AnalyzeDocumentFromUri` method and specify `DocumentAnalysisFeature.Formulas` in the `AnalyzeDocumentOptions`. The returned value is an `AnalyzeResult` object containing data about the submitted document.
+```C# Snippet:DocumentIntelligenceSampleHighResolutionExtraction
+```
+
+## Formula extraction
+
+## Font property extraction
+
+## Barcode property extraction
+
+## Language detection

```

Comment:
@johnna1314  pls reorganize your content with the structure

Reply:
updated.